COPYING

```diff
@@ -0,0 +1,7 @@
+Format-Specification: http://anonscm.debian.org/viewvc/dep/web/deps/dep5/copyright-format.xml?revision=279&view=markup
+Name: ceph-iscsi-config
+Maintainer: Jason Dillaman <dillaman@redhat.com>
+Source: http://github.com/ceph/ceph-iscsi-config
+
+Files: *
+License: GPL-3.0-or-later (see LICENSE)
```

Comment:
LICENSE and the existing spec file indicate this should be "GPL-3.0-only", unless I've missed something.

Reply:
Part of what this PR is trying to do is to clarify whether the license is GPL-3.0 "or later" or "only" 

It seemed reasonable to assume "or later" so I prepared the PR with that.  The spec file is changed to "or later", the COPYING file specifies "or later", and the LICENSE file says: 

```
Each version is given a distinguishing version number.  If the
Program specifies that a certain numbered version of the GNU General
Public License "or any later version" applies to it, you have the
option of following the terms and conditions either of that numbered
version or of any later version published by the Free Software
Foundation.
```

Reply:
IANAL, but my hope is that if we have "License: GPL-3.0-or-later" in COPYING, then the Program will be deemed to specify 'that a certain numbered version of the GNU General Public License "or any later version" applies to it.'